### DISCUSSION TITLE
Music Provider Feature Request Poll #1

### DISCUSSION THREAD

**Discussion by OzGav** [music-assistant/support#744] [Feature Polls]
The discussions have shown a number of people have ideas on what music provider they would like to see added next. These are not that easy to find and hard to let Marcel know what is popular in the Community. So have your say here!

If you would like to discuss these options in more detail then here are the threads:
Deezer https://github.com/music-assistant/hass-music-assistant/discussions/434
Apple Music https://github.com/music-assistant/hass-music-assistant/discussions/389
Tidal https://github.com/music-assistant/hass-music-assistant/discussions/432
Pandora https://github.com/music-assistant/hass-music-assistant/discussions/393
WebDAV https://github.com/music-assistant/hass-music-assistant/discussions/526
Emby https://github.com/music-assistant/hass-music-assistant/discussions/583
Amazon Music https://github.com/music-assistant/hass-music-assistant/discussions/626
Plex https://github.com/music-assistant/hass-music-assistant/discussions/482

**Comment by Dutchy-79** [comment]
I'd say apple music because my daughter uses it but I already have all I need.

**Reply by OzGav** [reply]
Cast your vote above then :-)

**Reply by Dutchy-79** [reply]
My vote would be "none because everything I use is already implemented"

**Comment by compujunk** [comment]
none of the above

**Comment by realroywalker** [comment]
You're missing soundcloud - that get's my vote 😆 https://github.com/music-assistant/hass-music-assistant/discussions/517

**Reply by OzGav** [reply]
Yeah sorry the poll was limited in the number of responses 😩

**Reply by WebfinityDesign** [reply]
I would like to see soundcloud too

**Reply by Bodge-IT** [reply]
I'd vote soundcloud or mixcloud.  My vote went to Plex but then I realised that doesn't really give me anything I don't already have.

**Reply by gieljnssns** [reply]
Soundcloud is available on the v2 beta. https://github.com/orgs/music-assistant/discussions/1160

**Comment by LukaszP2** [comment]
What about [Jellyfin](https://jellyfin.org/)?

**Reply by OzGav** [reply]
Thanks. This is being discussed here https://github.com/music-assistant/hass-music-assistant/discussions/407

**Comment by lzinga** [comment]
I am in for Pandora.

**Comment by ronaldvandervegt** [comment]
I would like Soundcloud, but as it wasnt there I voted for the only other usefull one to me: WebDav.

**Reply by OzGav** [reply]
There was a limit to the number of options I could have in the poll. There is an existing post on Soundcloud though so you can contribute there perhaps https://github.com/music-assistant/hass-music-assistant/discussions/517

**Reply by OzGav** [reply]
@ronaldvandervegt Soundcloud is available on the v2 beta. https://github.com/orgs/music-assistant/discussions/1160

**Comment by BristolBEAT** [comment]
Amazon Music is now free for Prime Members. Can we please implement!

**Comment by richardwonka** [comment]
Multiple spotify accounts would be nice

**Reply by OzGav** [reply]
In a few weeks v2 will be released which will provide some multi account support. Hopefully this might fulfill your wish…

**Reply by richardwonka** [reply]
Star! ⭐️

**Comment by BristolBEAT** [comment]
@OzGav Any irons in the fire for adding more music providers (my fingers personally crossed for Amazon Music).

**Reply by OzGav** [reply]
Yes I would like Amazon Music as well. Nothing concrete that I have heard. Marcel has said though that the new version being worked on will make it easier for people to add provider support. So hopefully that creates some activity with others contributing to the project.

**Comment by richardwonka** [comment]
Here’s a vote for soundcloud 😊

**Reply by OzGav** [reply]
@richardwonka Soundcloud is available on the v2 beta. https://github.com/orgs/music-assistant/discussions/1160

**Comment by SteveDinn** [comment]
It's not listed, but I'd love to see a **Subsonic**-API-based provider. I self-host a Subsonic server (Navidrome) where all my music is already indexed. It would be sweet if MA could read the artist, album, and track info from it and also stream from it.

**Comment by rriempie** [comment]
Apple Music please!

**Comment by DutchJaFO** [comment]
Would Audible count as a 'music provider' ?

I do have Plex, but the smb-provider has replaced that already. So unless that also includes access to the magic lists that Plex can create I really have no preference as with the exception of Audible all my current needs have been met.

**Reply by OzGav** [reply]
This is the thread for audible which you have previously replied to https://github.com/music-assistant/hass-music-assistant/discussions/862
